push-image.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+TAG=$(< VERSION)
+
+docker tag debify registry.tld/debify:$TAG
```

Comment:
This tag should be `registry.tld/conjurinc/debify`, to match the source repo. We've started doing this for our internal registry to make it easier to navigate.

Reply:
Ok. I just copied these commands from your `pipeline` branch. I'll fix the tags